src/canvas/widgets/battery_display.rs

```diff
@@ -153,8 +153,13 @@ impl Painter {
                         self.colours.high_battery_colour
                     }),
                 ]));
+                let state = match &battery_details.battery_duration {
+                    BatteryDuration::ToFull(_) => "Charging",
+                    BatteryDuration::ToEmpty(_) => "Consumption",
+                    _ => "Unknown",
```

Comment:
This also ends up being a bit confusing if you're at full and suddenly you get 
<img width="560" alt="image" src="https://user-images.githubusercontent.com/34804052/232595444-00220c31-3565-4ea2-bfe4-09bb453b8145.png">

Reply:
Sure, but you'll need to change `BatteryDuration` to include another status. Currently there are non that can represent `Full`.

Or, you can try and map against, `State` in the battery crate.
https://docs.rs/starship-battery/latest/starship_battery/enum.State.html

Reply:
Regardless, that's going to be too big of a change in this PR so I rather not do it.